CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ if (MSYS)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -DLOG4CPP_FIX_ERROR_COLLISION=1 -D__USE_W32_SOCKETS")
   add_definitions(-DOPT_TLS_OPENSSL)
 elseif (NOT APPLE)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -O3")
```

Comment:
cmake automatically applies optimizations based on build-type (CMAKE_BUILD_TYPE)
(at least in newer versions)

Reply:
ok thanks, for that reason i don't like cmake.